R/step-subset.R

```diff
@@ -135,6 +135,47 @@ select.dtplyr_step <- function(.data, ...) {
   step_group(out, groups)
 }
 
+#' @importFrom dplyr relocate
+#' @export
+relocate.dtplyr_step <- function(.data, ..., .before = NULL, .after = NULL) {
+  vars <- set_names(.data$vars, .data$vars)
+  to_move <- unname(tidyselect::eval_select(expr(c(...)), data = vars))
+  groups <- .data$groups
+  if (length(to_move) == 0) {
+    return(.data)
+  }
+  .before <- enquo(.before)
+  .after <- enquo(.after)
+  has_before <- !quo_is_null(.before)
+  has_after <- !quo_is_null(.after)
+  if (has_before && has_after) {
+    abort("Must supply only one of `.before` and `.after`.")
+  }
```

Comment:
Can you please follow https://style.tidyverse.org and put the `else` on the same line as the `}`?